sycl/include/CL/sycl/accessor.hpp

```diff
@@ -214,10 +218,30 @@ namespace sycl {
 template <typename DataT, int Dimensions = 1,
           access::mode AccessMode = access::mode::read_write,
           access::target AccessTarget = access::target::global_buffer,
-          access::placeholder IsPlaceholder = access::placeholder::false_t>
+          access::placeholder IsPlaceholder = access::placeholder::false_t,
+          typename PropertyListT = ONEAPI::accessor_property_list<>>
 class accessor;
 
 namespace detail {
+template <typename T>
+using IsPropertyListT = typename std::is_base_of<PropertyListBase, T>;
+
+template <typename T>
+using IsRunTimePropertyListT =
+    typename std::is_same<ONEAPI::accessor_property_list<>, T>;
```

Comment:
Are there no runtime properties available?

Reply:
Not yet.

Reply:
Before this patch, accessor constructors could accept property_list. To allow transparent conversions for user from property_list to some accessor_property_list, I had to make some differentiation between accessor_property_list with template parameters (aka constexpr parameters) and without (aka run time parameters). Runtime parameters do not require any template arguments, because they're stored in run time container.